src/frontend/packages/lib_components/src/common/index.tsx

```diff
@@ -68,6 +71,7 @@ export * from './TextInput';
 export * from './ProgressionBar';
 export * from './RetryUploadButton';
 
+export * from './TextTruncated';
```

Comment:
It exists a folder `Styled`, after I don't find the folder `Styled` very relevant neither.
What about created a folder `Text`, and put inside `TextInput` / `TextTruncated` / `TextAreaInput` ... ?